package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "react-use-click-away",
-  "version": "1.0.9",
+  "version": "2.0.0",
   "description": "a custom React effect for clicking away to close dropdowns",
   "scripts": {
     "build": "webpack",
     "dev": "webpack --watch",
     "prepare": "npm run build && npm version patch",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "main": "./lib/index.js",
+  "main": "./lib/index.ts",
```

Comment:
Hmm...I think `main` has to point to the file consumed when a user installs this package. So there's probably more to it then just this. I think we'd have to have a build step where we compile the TS -> JS in some `/dist` or `/out` dir and then publish that JS file.